node_modules/oae-core/upload/js/upload.js

```diff
@@ -478,9 +467,6 @@ define(['jquery', 'oae.core', 'jquery.fileupload', 'jquery.iframe-transport', 'j
                 }
 
                 // Disable editing on upload
-                // Note: the file input element can not be disabled, as that will cause IE9
-                // to drop it from the  DOM and not submit its file content to the server
-                $('#upload-modal *').not('input[type="file"]').prop('disabled', true);
```

Comment:
This shouldn't be taken out... What it's doing is disabling the upload modal when we're already in progress. We could change it to `$('#upload-modal *').prop('disabled', true);` though since we're no longer limited by IE9 constriction on `input[type="file"]`.